### ISSUE TITLE
Find/order a way to engrave or print labels for tubes

### ISSUE THREAD

**Issue by AHuffmyer** [RobertsLab/resources#1950]

**Comment by kubu4** [comment]
Pricing requested from:

https://tubewriter.com/

**Comment by AHuffmyer** [comment]
Could we suggest to SAFS as a departmental purchase??

**Comment by kubu4** [comment]
TubeWriter 300 (prints directly on tubes - no stickers/labels needed):

`$39,999`

Includes:

-  Four standard types of tube racks (e.g. 1.7mL, 2.0mL, 15mL, 50mL, etc)
- One year of ink cartridges - shipped every 90 days

**Comment by AHuffmyer** [comment]
That is pretty spendy... maybe would only make sense as a department purchase/community resource?

**Comment by AHuffmyer** [comment]
I spoke with a couple reps to get an idea of our options here before closing out the issue. Here are the options: 

**1) More expensive, but fanciest: TubeWriter 200 ($15,000) - TubeWriter**

Pros: 
- Will print directly on tubes using UV cured ink, compatible with freezer, fridge, liquid nitrogen, -80C storage, etc. 
- Easy to use - operates by adding desired labels to an Excel file
- Prints quickly (18-20 tubes printed in 2-3 min)
- Can print on cryovials, eppendorf tubes, PCR tubes
- Comes with 4 ink cartridges (see "Cons" below)

Cons: 
- Expensive
- Ink will expire in 90 days due to the UV curing process. If we don't use all the ink (enough for about 15K prints), it will go to waste. 
- We would get 4 ink cartidges (one every 90 days for the first year). After that, ink costs $800 per cartridge. 

**2) Cost effective: Brady M510 portable label printer ($914) - VWR**

Pros: 
- Inexpensive
- Flexible applications - could be used for any application and many types of labels are available 
- Compatible with cryo preservation labels 
- Hard wired or computer independent operation
- Portable, rechargeable battery 

Cons: 
- Requires purchase of label cartridges
- Does not directly print on tubes, requires printing on labels then adding those labels to tubes (still a lot easier than hand writing!) 

I have quotes for both options. @kubu4 what do you think? @sr320 would you like to move forward with either option? I am in favor of the Brady printer due to cost and our low to medium demand for labeling.

**Comment by AHuffmyer** [comment]
Brady labeler: https://www.bradyid.com/label-printers/portable/m510-portable-label-printer-laboratory-id-software-accessories-kit-pid-m510-lab?Ntt=m510-lab&el=products&el_text=M510-LAB  

TubeWriter 200 is brand new so its not on their website yet. Here is the next model up, the 300: https://tubewriter.com/tubewriter-300/ The 200 is smaller than this model.

**Comment by AHuffmyer** [comment]
I now have quotes for each system listed above (see pros and cons list above). 

@kubu4 what are your thoughts on the Brady label printer vs the TubeWriter 200 direct tube labeler? 

Brady Label printer with cryo labels: $914.34
TubeWriter 200: $15,834.14

**Comment by kubu4** [comment]
My personal preference would be TubeWriter, but there's a significant savings (obviously) with the other option.

The other "issue" I see with the Brady Label printer is that it seems like it only prints labels that can be wrapped around the sides of tubes. Personally, I'd prefer being able to have info on the _tops_ of the tubes so that we can see the labels when they're in boxes. Otherwise, we have to pull out each tube to see the labels.

**Comment by AHuffmyer** [comment]
The TubeWriter 200 would be nice. @sr320 do you think there are other labs that would be interested in using it as well? I just want to make sure the ink wouldn't go to waste before its expiration if we do not use it all. Lorenz said he was interested the other day...

**Comment by kubu4** [comment]
Yeah, the ink thing with the TubeWriter is concerning (and is _major_ drawback).

**Comment by AHuffmyer** [comment]
If there are other folks in the department that would use it, then it might make sense to do a joint purchase perhaps @sr320 ?

**Comment by AHuffmyer** [comment]
It seems that a tube labeler purchase is probably not a big priority for us at this time. If you agree, @sr320 and @kubu4 I can tell the vendors we won't be purchasing to close out those quote requests. 

If we do want to purchase one of these models I'm happy to submit a request.

**Comment by sr320** [comment]
hands down buy the Brady Label printer with cryo labels: $914.34  - no brainer. 

We can maybe buy Tube Writer in April also - lets see where budget / samples are.

**Comment by AHuffmyer** [comment]
Great, I'll move forward with the label printer through VWR and hold on the TubeWriter

**Comment by AHuffmyer** [comment]
Order submitted for Brady label system. I'm going to close this issue for now and we can revisit if needed.
